src/main/java/com/google/api/codegen/config/PackageMetadataConfig.java

```diff
@@ -400,24 +284,13 @@ public static PackageMetadataConfig createFromPackaging(
     return builder.build();
   }
 
-  private static Map<TargetLanguage, ReleaseLevel> mergeReleaseLevels(
-      Map<TargetLanguage, ReleaseLevel> lhs, ReleaseLevel thisReleaseLevel) {
-    if (thisReleaseLevel == null) {
-      return lhs;
-    } else {
-      Map<TargetLanguage, ReleaseLevel> result = new HashMap<>();
-      for (Map.Entry<TargetLanguage, ReleaseLevel> entry : lhs.entrySet()) {
-        // This just sets every language's release level to the release level of the current
-        // context. This is admittedly a little heavy-hadned, but it doesn't matter since
-        // we only run one language in an invocation of gapic-generator.
-        result.put(entry.getKey(), thisReleaseLevel);
-      }
-      return result;
-    }
+  private static ReleaseLevel mergeReleaseLevels(
```

Comment:
Can we remove this function entirely and use firstNotNull?

Reply:
Will do.